### DISCUSSION TITLE
群晖filestation添加webdav报错拒绝

### DISCUSSION THREAD

**Discussion by zhangle83** [AlistGo/alist#1019] [Q&A]
网页可以正常登录，filestation中添加webdev，对应输入ip和端口，路径为/dav,用户名密码为后台设置的webdev设置，一直报错连接被服务器拒绝

**Comment by zhangle83** [comment]
路径改成去了/，只填写dav或者为空也一样的报错

**Reply by zmy0516** [reply]
我路径不填就空着，是可以的

**Reply by miraclemie** [reply]
> 路径改成去了/，只填写dav或者为空也一样的报错

老哥解决了吗，会不会和黑群晖有关吗

**Reply by mt25cyb** [reply]
白群晖也一样

**Reply by mt25cyb** [reply]
白群晖测试成功，我用的是Docker安装Alist
-------- 下面是群晖File Station添加WebDAV的详细设置 --------
IP：127.0.0.1
端口：5244
路径：dav
账号：admin
密码：admin123
代码页：Unicode(UTF-8)
配置文件名称：Alist
-------- 上面是群晖File Station添加WebDAV的详细设置 --------
上面给了灵感，去尝试了下，路径改成去了/，只填写dav，不能为空，挂载成功。

**Reply by miraclemie** [reply]
试了一下可以了，感谢老哥

**Reply by owllen** [reply]
路径改成只填写dav，去掉/，测试可用了，感谢

**Reply by cguo2013** [reply]
上述各种方法都试过了, 还是报错连接被服务器拒绝, 是因为alist跟群晖的IP地址一样(端口不同) 导致的么?

**Reply by Westwall98** [reply]
试了上面的方法，还是会被拒绝，但是相同的配置Kodi可以访问，是不是群晖本身需要额外设置什么权限？
配置如下：
IP：127.0.0.1
端口：5244
路径：dav
账号：account
密码：password
代码页：Unicode(UTF-8)
配置文件名称：Alist

**Reply by ghostvarg** [reply]
我也是这个问题，用的是群晖套件，不知道怎么解决

**Reply by sinaamos** [reply]
一样碰到了，能添加进filestation，但无法连接，路径是是对的

**Reply by chotamaki** [reply]
应该是IP+dav/+你挂载网盘的名称（这个名称一定是英文，不能是汉语），这样应该可以了，比如我挂载的夸克网盘路径就是  http://IP:5244/dav/quark，不能直接用 http://IP:5244/dav，这样要么是报错，要么没反应，你可以试试我这个，我威联通这么挂载没什么问题，群晖大概率一样

**Reply by sinaamos** [reply]
@chotamaki 不行，还是挂不上，不知道是不是docker的网络问题

**Reply by KolinLoa** [reply]
把IP地址改成localhost，然后路径改成dav就可以了

**Reply by adamzhan9** [reply]
原来是/的问题

**Reply by jumoon2023** [reply]
@sinaamos  docker网络用 ”bridge”

**Reply by Mahuan06** [reply]
localhost + 去除/就可以了

**Comment by huangbinyu1987** [comment]
用localhost  把/去掉，只填dav就可以了。

**Reply by zhz-we** [reply]
还是不行。。。

![image](https://github.com/alist-org/alist/assets/42529417/af375c69-13ec-41dc-9251-ad82042c76df)

**Reply by jumoon2023** [reply]
路径应该填写dav，主机填写你的公网ip或者同一路由下的话可以是内网ip

**Reply by blackwing8121** [reply]
正解，感谢，已经解决了

**Comment by gjycode** [comment]
<img width="1606" alt="图片" src="https://github.com/user-attachments/assets/b8610134-fffb-4e2e-ab18-709ed17eb61e" />
如果使用的是群晖套件，遇到【连接被服务器拒绝】，直接到Alist管理中，把用户的功能权限勾选上就可以了，还有前文的dav不要加/

**Reply by philipzb** [reply]
谢谢大佬，管用
